test/e2e/tests/errors.spec.js

```diff
@@ -73,7 +135,43 @@ describe('Sentry errors', function () {
         },
       );
     });
-    it('should send error events', async function () {
+
+    it('should NOT send error events in the UI', async function () {
+      await withFixtures(
+        {
+          fixtures: new FixtureBuilder()
+            .withMetaMetricsController({
+              metaMetricsId: null,
+              participateInMetaMetrics: false,
+            })
+            .build(),
+          ganacheOptions,
+          title: this.test.title,
+          failOnConsoleError: false,
+          testSpecificMock: mockSentryTestError,
+        },
+        async ({ driver, mockedEndpoint }) => {
+          await driver.navigate();
+          await driver.findElement('#password');
+          // Erase `getSentryState` hook, simulating a "before initialization" state
+          await driver.executeScript(
+            'window.stateHooks.getSentryState = undefined',
+          );
+
+          // Wait for Sentry request
+          await driver.delay(3000);
```

Comment:
Non-blocking nit but we may want to stick to known `delay` constants rather than an arbitrary number.  Can be done in a follow up though.

Reply:
yeah good call. This was copied from pre-existing tests in this file, and they should all be using a matching delay time, so I'll leave this for a follow-up.